Simplenote/src/main/java/com/automattic/simplenote/utils/ThemeUtils.java

```diff
@@ -46,7 +46,7 @@ public static void setTheme(Activity activity) {
                 }
             }
         }
-
+        
```

Comment:
```suggestion

```

This looks like some unnecessary spaces might have been accidentally added.  We can remove them.